flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/operators/join/deltajoin/StreamingDeltaJoinOperator.java

```diff
@@ -143,9 +143,10 @@ public class StreamingDeltaJoinOperator
 
     private final boolean[] isInputEnded;
 
-    // ---------------------------- Metrics -----------------------------------
```

Comment:
nit: why did we move this line , it seemed to fine where it was - I assume `totalInflightNum ` is a metric

Reply:
The `totalInflightNum` is not only a metric but also related to operator logic such as `tryProcess`, `allInflightFinished`, etc.